projects/core/src/icon/icon.stories.ts

```diff
@@ -67,12 +68,100 @@ class AllIcons extends LitElement {
   }
 }
 
+@customElement('demo-icons-collection')
+class CollectionIcons extends LitElement {
+  @state() icons: string[] = [];
+
+  private _collection: string;
+  get collection(): string {
+    return this._collection;
+  }
+
+  @property({ type: String })
+  set collection(val: string) {
+    this._collection = val;
+  }
```

Comment:
This can be simplified to just `@property({ type: String }) collection: string;`. No need for accessors and a backing property.